### PR TITLE
Documentation for monorepo

### PR DESCRIPTION
**Pull request by musangowope**

### CODE REVIEW

File: core/scripts/custom-elements/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@laws-africa/law-widgets/components",
+  "name": "@lawsafrica/law-widgets/components",
   "version": "0.0.0",
   "description": "Ionic Components exported as custom elements, extending HTMLElement.",
```
Comment (longhotsummer) [inline core/scripts/custom-elements/package.json:4]: Change description

File: law-widget-styles/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "scripts": {
+    "test": "jest",
+    "build": "sass --style compressed scss/law-widget-styles.scss css/law-widget-styles.min.css && sass scss/law-widget-styles.scss css/law-widget-styles.css"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/laws-africa/law-widget-styles.git"
+  },
+  "author": "Greg Kempe <greg@laws.africa>",
+  "license": "MIT",
```
Comment (longhotsummer) [inline law-widget-styles/package.json:19]: Change to match the law-widgets license (LGPL 3.0)

**Review by longhotsummer — CHANGES_REQUESTED** [review]
This looks great, just some minor comments. Then we can merge  into the mono-repo branch and prep that for final merge and release.

File: core/scripts/custom-elements/package.json (hunk 1 above)
Comment (musangowope) [reply core/scripts/custom-elements/package.json]: done

File: core/.prettierrc.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  ...require('@musangowope/prettier-config'),
+  ...require('@lawsafrica/prettier-config'),
```
[the remark this answers is not in the document]
Comment (musangowope) [reply core/.prettierrc.js]: Good catch. Changed it to `...require('@ionic/prettier-config')`. That's what was in the ionic repo. They have it as a devDependency as do we in our repo. I think, lets just use their _opinionated_ configuration.

File: law-widget-styles/package.json (hunk 2 above)
Comment (musangowope) [reply law-widget-styles/package.json:19]: done

File: law-widget-styles/README.md [hunk 4]
```diff
@@ -19,21 +19,21 @@ Second, include the CSS as described below.
 
 Use the assets directly from the [jsDelivr](https://www.jsdelivr.com) CDN:
 
-    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/laws-africa/indigo-web@8.1.0/css/indigo-web.min.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/laws-africa/law-widget-styles@8.1.0/css/law-widget-styles.min.css">
```
[the remark this answers is not in the document]
Comment (musangowope) [reply law-widget-styles/README.md]: done

File: law-widget-styles/package.json (hunk 2 above)
Comment (musangowope) [reply law-widget-styles/package.json]: done

File: law-widget-styles/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (longhotsummer) [reply law-widget-styles/README.md]: this still isn't right because it's using the github package repo, not npm, but we'll fix that later.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**